### PR TITLE
BUG

### LINKED ISSUE #9: Adding titles to multiple columns
Regression: 
Adding titles to multiple columns will cause the columns to switch

### PR DESCRIPTION
**Pull request by ChandlerBaskins**
Resolved issue when editing titles causes columns to move.
Changed title elements to proper inputs

### CODE REVIEW

**Comment by ChandlerBaskins** [comment]
CC @brittneypostma

File: src/components/column.svelte [hunk 1]
```diff
@@ -27,16 +29,11 @@
   }}
   on:dropped
 >
-  <header
-    class="text-sm font-thin h-5 border-b-2 border-blue-600 overflow-hidden cursor-text"
-    on:click={() => (isTitleEditing = true)}
+
+
+  <input on:mousedown|stopPropagation on:blur="{(e) => editColumnName(id,e.target.value)}" value={title} type="text"
```
Comment (sz3ry) [inline src/components/column.svelte:34]: Stops handling mousedown event from input element so it wont be passed to the drag handler

**Review by sz3ry — APPROVED** [review]
Thank you ! :)

**Comment by sz3ry** [comment]
Last commit fixes definitely #9

File: src/components/column.svelte [hunk 2]
```diff
@@ -50,20 +47,7 @@
       </p>
     </slot>
   </div>
-  <div class="flex justify-between w-full">
```
Comment (sz3ry) [inline src/components/column.svelte:53]: Removed these buttons, it was only for testing purposes of reorder

File: src/store/board.store.js [hunk 3]
```diff
@@ -45,12 +45,13 @@ export function reverseState(){
  * @param {number} newIndex New index of a Column
  */
 export function changePosition(oldIndex, newIndex){
-    
-    board.update(state=>{
-        const newState = state.filter(arrayElement=>arrayElement.id !== state[oldIndex].id)
-        newState.splice(newIndex, 0, state[oldIndex])
-        return newState
-    })
+    if(oldIndex!==newIndex && newIndex >= 0 ){
```
Comment (sz3ry) [inline src/store/board.store.js:48]: should it be more type checked to be sure passed indexes are numbers ?

**Review by brittneypostma — APPROVED** [review]
This looks good @ChandlerBaskins, I'll dive in more when I pull down the code.

**Pull request merged**